include/rocksdb/table_properties.h

```diff
@@ -239,6 +240,10 @@ struct TableProperties {
   // 0 means not exists.
   uint64_t external_sst_file_global_seqno_offset = 0;
 
+  // Offset where the "tail" part of SST file starts
+  // "Tail" refers to all blocks after data blocks till the end of the SST file
+  uint64_t tail_start_offset = 0;
```

Comment:
The motivation for adding this appears to be for SST ingest/import, which is reasonable.

Reply:
Yes, mostly for SST ingest/import/repair